quick-start.pod

```diff
@@ -950,6 +963,17 @@ In this case, we first indicate that we want to do a substitution with
 C<s>, then we show which word or letter we want to change, and then
 we write the word or letter that is written instead.
 
+=head4 Exercise
+
+Perl's motto is "There is more than one way to do it". In this exercise,
+change this sentence so that it prints the motto:
+
+    my $string = "There is only one way to do it";
+    $string =
+    say $string;
+    __TEST__
+    like($output, qr/There is more than one way to do it$/, "It should print Perl's motto");
```

Comment:
OK, I found that `is` doesn't work unless `\n` is added to the text as `say` always appends the new line.